components/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import React, {PropsWithChildren, ReactNode, useCallback} from 'react';
+import {TouchableOpacity} from 'react-native';
+import {Box, IBoxProps, Divider, VStack} from 'native-base';
+
+export interface CardProps extends IBoxProps {
+  header?: ReactNode;
+  onPress?: () => void;
+  borderRadius?: number;
+  borderColor?: string;
+}
+
+export const Card: React.FunctionComponent<PropsWithChildren<CardProps>> = ({header, onPress, borderColor, borderRadius, children, ...boxProps}) => {
+  const pressHandler = useCallback(() => onPress?.(), [onPress]);
+
+  return (
+    <Box {...boxProps}>
+      <TouchableOpacity onPress={pressHandler}>
+        <Box bg="white" borderWidth="2" borderRadius={borderRadius ?? 8} borderColor={borderColor ?? 'light.200'} p="4">
```

Comment:
Are we sure that we're not going to need to overwrite anything else on this nested `Box` (and, if not, is there a pattern here that doesn't require us to expose each additional prop as an optional parameter for the `Card`?)

Reply:
Not sure if we'll want more control than this - I wouldn't want to spend too much time implementing solutions until there's a need. (I'm actually not certain the outer box is even needed, we might want to just pass through Box properties onto the inner box and remove the outer one)

Reply:
Sounds good!